lua/treesitter-context.lua

```diff
@@ -406,7 +406,9 @@ end
 
 local function win_close(winid)
   if winid ~= nil and api.nvim_win_is_valid(winid) then
-    api.nvim_win_close(winid, true)
+    pcall(function()
+      api.nvim_win_close(winid, true)
+    end)
```

Comment:
```lua
pcall(api.nvim_win_close, winid, true)
```

And add a reference to this PR.

The proper solution is probably just to call it via `vim.schedule`.

Reply:
`vim.schedule(api.nvim_win_close(winid, true))` produces the same error, not sure why.